### PR TITLE
Terry

### PR DESCRIPTION
**Pull request by yishengjin1413**
use emoji only

### CODE REVIEW

File: MarkdigEngine.Extensions/Emoji/EmojiParser.cs [hunk 1]
```diff
@@ -0,0 +1,963 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Markdig.Helpers;
+using Markdig.Parsers;
+
+namespace MarkdigEngine.Extensions
+{
+    public class EmojiParser : InlineParser
```
Comment (xoofx) [inline MarkdigEngine.Extensions/Emoji/EmojiParser.cs]: Ideally, this should be done in Markdig, just to provide an option to allow emoji and/or smileys. Duplicating the code here is not super good. Will update Markdig accordingly to provide this option.

Comment (qinezh) [reply MarkdigEngine.Extensions/Emoji/EmojiParser.cs]: @xoofx It sounds better, thanks a lot.

Comment (yishengjin1413) [reply MarkdigEngine.Extensions/Emoji/EmojiParser.cs]: @xoofx Sure. That would be great, will remove the duplicated code once you update the markdig.

Comment (xoofx) [reply MarkdigEngine.Extensions/Emoji/EmojiParser.cs]: In 0.14.3, this is now accessible through [`EmojiExtension`](https://github.com/lunet-io/markdig/blob/07a2980d5b272c4a23b22f39ffc23975311d0c8e/src/Markdig/Extensions/Emoji/EmojiExtension.cs#L20) (or directly on the `EmojiParser`

Comment (yishengjin1413) [reply MarkdigEngine.Extensions/Emoji/EmojiParser.cs]: Thanks @xoofx , I used your new extension in this [PR](https://github.com/docascode/markdown/pull/62).

**Pull request merged**